doc/rst/source/grdimage_notes.rst_

```diff
@@ -11,6 +11,18 @@ linear projection, or (b) use :doc:`grdview` **-Ts** instead.
 
 .. include:: explain_grdresample.rst_
 
+Imaging Categorical Grids
+-------------------------
+
+Geographical categorical grids have values at the nodes that should not
+be interpolated (i.e., categories 4 and 5 should never be averaged to give
+a new category 4.5).  However, imaging such grids using map projections
+requires a resampling onto an equidistant Cartesian lattice that usually
+will result in such blending.  We therefore override any **-n** setting you
+have chosen (perhaps implicitly) with **-nn+a** that turns *on* nearest neighbor
+gridding and turns *off* anti-aliasing.  Alternatively, use :doc:`grdview` **-T**
```

Comment:
If I understand it correctly, it only happens when a categorical CPT is used, right? I think we should also mention this point, otherwise it's unclear to me how GMT knows a grid is categorical or not.

Reply:
Yes, will do that.  In fact already doing that in a PR for grdview...

Reply:
Now updated